src/modules/common/__fixtures__/mockCommandParametersSchemas.js

```diff
@@ -0,0 +1,285 @@
+// eslint-disable-next-line import/prefer-default-export
+export const mockCommandParametersSchemas = {
```

Comment:
Snapshots are helpful in this kind of scenarios. Although I don't ask you to change it.

Reply:
@reyraa Noted